eng/common/scripts/SemVer.ps1

```diff
@@ -78,13 +78,10 @@ class AzureEngSemanticVersion : IComparable {
 
       if ($null -eq $matches['prelabel'])
       {
-        # artifically provide these values for non-prereleases to enable easy sorting of them later than prereleases.
-        $this.PrereleaseLabel = "zzz"
```

Comment:
This will break https://github.com/Azure/azure-sdk-for-go/blob/main/eng/scripts/Update-ModuleVersion.ps1#L43 so we will need to submit a fix.

Reply:
agreed.  

There are several copycat implementations in both typescript and powershell that are only used for version comparison or metadata and don't do increment.   It would be nice to have them no longer return artificial prerelease info as part of their parse.

Reply:
Fixed in -go repo